source/tutorials/metadata.rst

```diff
@@ -5,40 +5,42 @@ Metadata in QIIME 2
 
 Metadata provides the key to gaining biological insight from your raw data. In QIIME 2, sample metadata includes technical details, such as the DNA barcodes that were used for each sample, or descriptions of the samples, such as which subject, time point and body site each sample came from in a human microbiome time series. Feature metadata is often a feature annotation, such as the taxonomy assigned to a sequence variant. Sample and feature metadata are used by many methods and visualizers in QIIME 2.
 
-.. qiime1-users:: In QIIME 1, metadata (also known as the *metadata mapping file*) was a user-defined TSV file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata via a TSV file or by viewing QIIME 2 artifacts as metadata. Examples of both are presented in the following sections. Begin by creating a directory to conduct the work in:
```

Comment:
Oops, that was almost certainly copy-paste-errors-dot-com. Good catch @jairideout!